Makefile

```diff
@@ -25,7 +25,7 @@ DID_ELEMENT_SIDETREE_REQUEST_URL ?= https://element-did.com/api/v1/sidetree/requ
 SANDBOX_CLI_IMAGE_NAME       ?= trustbloc/sandbox-cli
 
 # TrustBloc core k8s deployment scripts https://github.com/trustbloc/k8s
-TRUSTBLOC_CORE_K8S_COMMIT=360545627756c03b72764e2be7b6a43779bb41d1
+TRUSTBLOC_CORE_K8S_COMMIT=574e5903199d86f3d7119208b8b50c56571dc552
```

Comment:
Can we update this to latest version https://github.com/trustbloc/k8s/commit/fe0aff36c639074b63f8bdf3e6dca04f05c63162 ?